openlibrary/plugins/openlibrary/lists.py

```diff
@@ -353,6 +353,23 @@ def POST(self, user_key: str | None, list_key: str | None = None):  # type: igno
             return safe_seeother(list_record.key)
 
 
+class lists_add_account(delegate.page):
+    path = r"/account/lists/add.*"
+
+    def GET(self):
```

Comment:
We can avoid the permission_denied by using the help `require_login` decorate, eg 
```suggestion
    @require_login
    def GET(self):
```

See https://github.com/internetarchive/openlibrary/blob/8c92c3ca40e56f8cd4d171561c1a89d66c8bf4b2/openlibrary/plugins/upstream/account.py#L846-L847